bin/pd2pg

```diff
@@ -75,25 +75,28 @@ class PG2PD
   end
 
   # Adds users to user_schedule table associated with given schedule.
-  def user_schedules_to_db(items, schedule_id)
+  def user_schedules_to_db(schedule_items)
     columns = [:id, :user_id, :schedule_id]
-    records = items.map do |i|
-      ["#{i['id']}_#{schedule_id}",
-       i['id'],
-       schedule_id]
+    all_records = []
+    schedule_items.each do |s|
+      users = get_bulk(:users,
+                                "schedules/#{s['id']}/users",
```

Comment:
This looks weirdly indented - are these tabs?

Reply:
Fixed!